src/publisher/message-queues.ts

```diff
@@ -111,6 +111,7 @@ export abstract class MessageQueue extends EventEmitter {
       }
       spanMessages.forEach(m => {
         tracing.PubsubSpans.updatePublisherTopicName(m.parentSpan!, topic.name);
```

Comment:
Technically not part of this PR but why is this necessary? It looks like the same information is already passed when `createPublisherSpan` is called.

Reply:
Mentioned over chat, but the trouble is that, since everything is async, it's possible that the code could get to the part that sets the initial span name before the project ID is resolved. In that case, the project ID will become `{{project}}` in the final span, which isn't great. The project ID must be resolved by this point, so this just makes sure the right names are in place.